### PR TITLE
[ft-benchmark] add using of db cli

### PR DESCRIPTION
**Pull request by MCJOHN974**
This code will take collected data and send it to ft-benchmark db using CLI

### CODE REVIEW

**Comment by MCJOHN974** [comment]
WIP

**Comment by MCJOHN974** [comment]
This is ready for review

File: scripts/ft-benchmark-data-sender.py [hunk 1]
```diff
@@ -85,45 +87,25 @@ def get_commit() -> tuple[str, datetime]:
...
-                    %(num_shards)s, %(num_unique_users)s, %(size_state_bytes)s, 
-                    %(tps)s, %(total_transactions)s
-                )
-                """)
-            cursor.execute(insert_query, data)
-            conn.commit()
+    chdir(os.path.expanduser("~/nearcore/benchmarks/continous/db/tool"))        
+    json.dump(data, open("tmp.json", "w"))
+    subprocess.run("cargo run -p cli -- insert-ft-transfer tmp.json", shell=True)
+    subprocess.run("rm tmp.json", shell=True)
```
Comment (akashin) [inline scripts/ft-benchmark-data-sender.py:94]: You can use a [tempfile](https://docs.python.org/3/library/tempfile.html#examples) for this, something along the lines:

```python
with tempfile.NamedTemporaryFile() as fp:
    json.dump(data, fp)
    fp.close()
    subprocess.run(f"cargo run -p cli -- insert-ft-transfer {fp.name}", shell=True)
```

**Review by akashin — APPROVED** [review]

**Review by mooori — APPROVED** [review]
Usage of the db cli LGTM.

**Pull request merged**